Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -983,6 +983,70 @@ ExceptionOr<void> LocalDOMWindow::postMessage(JSC::JSGlobalObject& lexicalGlobal
     return { };
 }
 
+ExceptionOr<void> LocalDOMWindow::postMessageFromRemote(JSC::JSGlobalObject& lexicalGlobalObject, std::optional<WebCore::SecurityOriginData> target, const WebCore::MessageWithMessagePorts& message)
```

Comment:
This always returns void.  Its return value is never used.  Make it just return void.